composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">= 7.1",
         "mustache/mustache": "^2.12",
         "ralouphie/mimey": "^2.0",
-        "symfony/yaml": "^2.3||^3.3",
+        "symfony/yaml": "^2.3||^3.3 || ~4.0",
```

Comment:
Two small tweaks:
1. Should be consistent with the other constraints, so `^` not `~`
2. As long as you're updating this line, could you fix the spacing around the other `||` as well so it is also consistent with `zendframework/zend-stdlib`

Reply:
Hi  already install a lasted version of `zendframework/zend-stdlib`your comment I fixed